class.jetpack-cli.php

```diff
@@ -1447,7 +1447,8 @@ function jetpack_cli_are_you_sure( $flagged = false, $error_msg = false ) {
 	}
 
 	if ( ! $flagged ) {
-		$prompt_message = __( 'Are you sure? This cannot be undone. Type "yes" to continue:', '"yes" is a command.  Do not translate that.', 'jetpack' );
+		/* translators: Don't translate the word yes here. */
+		$prompt_message = __( 'Are you sure? This cannot be undone. Type "yes" to continue:', 'jetpack' );
```

Comment:
Looks like there was a mistake - it should be `_x(`, not `__(` because of the second parameter.

Reply:
Thanks. 👍 I used `/* translators: ... */` -comment above it instead but I switched both lines to use `_x()` now for consistency with the rest of the file.